background.ts

```diff
@@ -1000,3 +1004,19 @@ function isRoot(): boolean {
 
   return os.userInfo().uid === 0;
 }
+
+async function runRdctlSetup(newSettings: settings.Settings): Promise<void> {
+  // don't do anything with auto-start configuration if running in development
+  if (isDevEnv) {
+    return;
+  }
+
+  const rdctlPath = path.join(paths.resources, os.platform(), 'bin', 'rdctl');
+  const args = ['setup', `--application.auto-start=${ newSettings.autoStart }`];
+
+  if (newSettings.debug) {
+    args.push('--verbose=true');
```

Comment:
I don't think this option is supported for `rdctl setup`

Reply:
:+1: will remove.